_includes/master/reqs-azure.md

```diff
@@ -0,0 +1,46 @@
+## About {{site.prodname}} on Azure
+
+Azure does not support {{site.prodname}} networking. Use one of the following
+options for networking in an Azure cloud.
+
+- **Azure user-defined routes**: This option provides networking without overlays.
+  Disable {{site.prodname}} networking by setting `CALICO_NETWORKING_BACKEND` to `none` 
+  in `{{site.nodecontainer}}`. (Also called "policy-only mode".) Refer to 
+  [Configuring calico/node](../../../../reference/node/configuration) and [Azure user-defined routes](#azure-user-defined-routes) for more information. {% if include.orch == "Kubernetes" %}You can
+  find a sample manifest in the [Kubernetes API datastore](../../installation/hosted/kubernetes-datastore/#policy-only) installation guide.
+
+- **flannel**: Refer to the [flannel installation guide](../../installation/hosted/canal) 
+  for specific instructions and a manifest. This option does use overlays.
+{% endif %}
+
+- **Azure CNI plug-in**: Configure {{site.prodname}} to use the 
+  [Azure CNI plug-in](https://github.com/Azure/azure-container-networking/blob/master/docs/cni.md)
```

Comment:
This needs to say "Azure CNI IPAM plug-in" - even so, I suspect most users won't be able to piece together how to do this just from that link. I guess it's the best we've got right now though.

Reply:
Yes, it is the best that we have at present, unfortunately.